packages/bodiless-documentation/doc/Development/Guides/BuildingSites/TypographySetup/TailwindGuide.md

```diff
@@ -23,128 +22,145 @@ Adding custom styling can be done by editing `site.tailwind.config.js`, followin
 documentation](https://tailwindcss.com/docs/configuration).
 
 If your site doesn't have a Tailwind configuration file in its root directory, then your site will
-use all of Tailwind's default settings as well as packages that include tailwind.  Bodiless has a mechanism that will merge all the tailwind configuration files coming from all packages and produce a single css file.
-
-?> **Important**  If the classes are duplicated in the site and package, SITE's css will always win.
-
-## Determine whether to use Extend vs Replace
-
-Tailwind allows replacing or extending the settings. For the majority of cases you want to
-[extend](https://tailwindcss.com/docs/theme/#extending-the-default-theme).  In cases like adding a new margin or padding custom definition 
-you want to add this definition to existing set and extend will do this.  If there are cases where you would want to restrict and totally redefine the settings, you would not use extend.  The one suggested case would be colors to restrict the color set so generic tailwind colors are not available and accidentally used.
-
-  ```
-  theme: {
-    colors: {
-      brand_blue: '#004c97',
-      brand_lightblue: '#017eb3',
-      brand_mediumblue: '#009cde',
-    },  
-    extend: {
-      padding: {
-        'custom_xs': '.75rem',
-      }      
+use all of Tailwind's default settings as well as packages that include Tailwind. Bodiless has a
```

Comment:
This section and http://localhost:8005/___docs/#/Development/Guides/BuildingSites/TypographySetup/TailwindGuide?id=tailwind-purge-whitelist-auto-discovery is kind of the same, we should merge them.  

We want to call out that Tailwind says to create a tailwind.config.js. but Bodiless system, we are using site.tailwindo.config.js in packages and sites level.  There is a custom. build mechanism to discover all packages' and sites' Tailwind config files look for this file name ('site.tailwind.confog.js') and combines them into one single Tailwind file during the build process.